style.css

```diff
@@ -0,0 +1,105 @@
+body {
+    background-color: #172b4d;
+}
+.container {
+    margin: 0 auto;
+    width: 315px;
+    height: 4210px;
+    background-color: #fff;
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+    padding: 30px;
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
- [ ] Good job making most of the required changes @francis450, But there is a minor detail you need to address before completing this milestone. The thing is we need to have a design as close as possible to the Figma design and as part of this your decoration images need to be aligned and styled to look like the one on the Figma design. Please make sure your decoration image is stretched to 100% width of the screen and not repeated.

### Your design
![Screenshot 2023-01-13 230643](https://user-images.githubusercontent.com/69077061/212410878-a9c1e4d1-5ece-4ecd-84f7-74ff8ad60323.png)

### Figma design 
![Screenshot 2023-01-13 230832](https://user-images.githubusercontent.com/69077061/212410788-f7f83228-0be4-4ec2-9e7c-658cf0290176.png)